plugin-api/src/test/java/org/sonar/api/batch/scm/BlameLineTest.java

```diff
@@ -35,8 +35,8 @@ public void testBlameLine() {
     assertThat(line1.author()).isEqualTo("foo");
     assertThat(line1.date()).isEqualTo(date);
     assertThat(line1.revision()).isEqualTo("1");
-    assertThat(line1.toString()).isNotEqualTo("foo");
-    assertThat(line1.toString()).contains("revision=1,author=foo");
```

Comment:
This was the only non-import change needed, the field order in `ToStringBuilder` changed, but it shouldn't have any impact.